neptune/alpha/internal/containers/disk_queue.py

```diff
@@ -138,43 +138,54 @@ def wait_for_empty(self, seconds: Optional[float] = None) -> None:
         self._event_empty.wait(seconds)
 
     def ack(self, version: int) -> None:
-        self._last_ack_file.write(version)
-        log_versions = self._get_all_file_versions()
+        log_versions = self._get_all_log_file_versions()
         for i in range(0, len(log_versions) - 1):
             if log_versions[i + 1] <= version:
                 os.remove(self._get_log_file(log_versions[i]))
             else:
                 break
+        self._remove_old_blob_files(version)
+        self._last_ack_file.write(version)
 
     def is_empty(self) -> bool:
         return self.size() == 0
 
     def size(self) -> int:
         return self._last_put_file.read_local() - self._last_ack_file.read_local()
 
+    def _get_blob_file(self, index: int) -> str:
+        return "{}/data-{}.blob".format(self._dir_path, index)
```

Comment:
1) Let's make a dedicated directory for blobs.
2) In future there can be more then one file for a signle operation (file sets, image series) so we should create a directory per operation.
3) It is not a good idea to keep everything in single directory if we don't know how many entries we can expect to be there. W need some more complex hierarchy of directories.